core/test/src/test_cpu.cpp

```diff
@@ -1194,12 +1194,19 @@ TEST_F(CpuTest, cli) {
 }
 
 TEST_F(CpuTest, rts) {
+    registers.pc = 0x1234;
     stage_instruction(RTS);
     registers.sp = 0x0A;
     expected.sp = 0x0C;
     expected.pc = 0xDEAD + 1;
 
-    EXPECT_CALL(mmu, read_word(kStackOffset + 0x0B)).WillOnce(Return(0xDEAD));
+    // Dummy read
+    EXPECT_CALL(mmu, read_byte(0x1235));
```

Comment:
This needs an InSequence if the order matters.

Reply:
Updated!